CHANGELOG.md

```diff
@@ -1,36 +1,79 @@
 # Rojo Changelog
 
 ## Unreleased Changes
-* Internally for themes, we use `Color3.fromHex` now instead of custom-made solution ([#761]).
-* On failing to open a file from Roblox Studio, it is now logged appropriately instead of being ignored ([#745]).
-* Significantly improved performance of `rojo sourcemap`. ([#668])
-* Fixed the diff visualizer of connected sessions. ([#674])
-* Fixed disconnected session activity. ([#675])
-* Skip confirming patches that contain only a datamodel name change. ([#688])
+### Additions
+* Added support for `.toml` files to `$path` ([#633])
+* Significantly improved performance of `rojo sourcemap` ([#668])
+* Add `plugin` flag to the `build` command that outputs to the local plugins folder ([#735])
+* Added `Terrain` classname inference, similar to services ([#771])
+
+	`Terrain` may now be defined in the projects without using `$classname`:
+	```json
+	"Workspace": {
+		"Terrain": {
+			"$path": "path/to/terrain.rbxm"
+		}
+	}
+	```
+
+* Added support for `Terrain.MaterialColors` ([#770])
+
+	`Terrain.MaterialColors` is now represented in projects in a human readable format:
+	```json
+	"Workspace": {
+		"Terrain": {
+			"$path": "path/to/terrain.rbxm"
+			"$properties": {
+				"MaterialColors": {
+					"Grass": [10, 20, 30],
+					"Asphalt": [40, 50, 60],
+					"LeafyGrass": [255, 155, 55]
+				}
+			}
+		}
+	}
+	```
+
+* Added better support for `Font` properties ([#731])
+
+	`FontFace` properties may now be defined using implicit property syntax:
+	```json
+	"TextBox": {
+		"$className": "TextBox",
+		"$properties": {
+			"FontFace": {
+				"family": "rbxasset://fonts/families/RobotoMono.json",
+				"weight": "Thin",
+				"style": "Normal"
+			}
+		}
+	}
+	```
+
+* Add Confirmation Behavior setting ([#774])
```

Comment:
This feels like it will cut off questions about this particular item
```suggestion
* Addded a setting to control patch confirmation behavior ([#774])
```

Reply:
I went ahead and added an explanation too